cmd/mattermost/commands/command_test.go

```diff
@@ -169,3 +169,53 @@ func TestDeleteCommand(t *testing.T) {
 		assert.Error(t, th.RunCommand(t, "command", "delete", "invalid"))
 	})
 }
+
+func TestModifyCommand(t *testing.T)  {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	team := th.BasicTeam
+	user := th.BasicUser
+	th.LinkUserToTeam(user, team)
+
+	originalCommand := &model.Command{
+		DisplayName: "dn_" + model.NewId(),
+		Method:      "G",
+		TeamId:      team.Id,
+		Username:    user.Username,
+		CreatorId:   user.Id,
+		Trigger:     "trigger_" + model.NewId(),
+		Description :"old description",
+		URL : "http://localhost:8000/modify-command",
+		IconURL : "original icon",
+		AutoComplete : false,
+	}
+	modifiedTitle := "modified title"
+	modifiedDescription := "modified description"
+	modifiedTrigger := "modified trigger"
+	modifiedURL := "http://localhost:8000/modified-command"
+	modifiedIcon := "modified icon"
+	modifiedUsername := "modified username"
+	modifiedCreator := "modified creator"
+
+	th.AddPermissionToRole(model.PERMISSION_MANAGE_SLASH_COMMANDS.Id, model.TEAM_USER_ROLE_ID)
+
+	t.Run("original command", func(t *testing.T) {
+		command, err := th.App.CreateCommand(originalCommand)
+		require.Nil(t, err)
+
+		commands, err := th.App.ListTeamCommands(team.Id)
+		require.Nil(t, err)
+		assert.Equal(t, len(commands), 1)
+
+		th.CheckCommand(t, "command", "modify", command.Id, "--title","--description", modifiedDescription,
+			modifiedTitle,"--trigger-word",modifiedTrigger,"--url", modifiedURL, "--creator", modifiedCreator,
+			"--response-username", modifiedUsername, "--icon", modifiedIcon, "--autocomplete", "--post")
+
+		commands, err = th.App.ListTeamCommands(team.Id)
+		require.Nil(t, err)
+		modifiedCommand := commands[0]
+		assert.Equal(t, len(commands), 1)
+		assert.Equal(t,"modifiedtrigger", modifiedCommand.Trigger)
```

Comment:
Probably I would add more asserts here to be sure everything is checked properly.